### ISSUE TITLE
Solid Notifications: add PATCH-like activities for resources

### ISSUE THREAD

**Issue by srosset81** [assemblee-virtuelle/activitypods#322]
Currently if we listen to a resource, we only get a `Update` activity and we need to refech the whole resource.

If we received a `Add > Relationship` or `Remove > Relationship` with the subjects, predicates and objects, we could update the resource locally without needing to refetch it entirely.

Suggested by @nikoPLP

**Comment by nikoPLP** [comment]
yes, something similar to what was done 3 years ago here https://github.com/assemblee-virtuelle/semapps/pull/987
